### DISCUSSION TITLE
Dịch dropout rate

### DISCUSSION THREAD

**Discussion by quangvnai** [mlbvn/handson-ml2-vn#114] [Terminology] [answered]
Từ này nên dịch là gì cho hợp lý mọi người nhỉ?
1. tỉ lệ dropout (ý kiến cá nhân, thấy cách này thích hợp hơn cách dưới)
2. giá trị dropout

**Comment by thanhcsf** [comment, marked as answer]
Mình chọn tỷ lệ dropout.
